resources/js/Shared/Layout/Nav.vue

```diff
@@ -52,6 +56,7 @@ const navigation = [
 function logout() {
   router.post('/logout')
   isMobileMenuOpened.value = false
+  toast.success(__('You have logged out successfully.'))
```

Comment:
Add `onSuccess` callback and move stuff here. Otherwise it will be shown even if logout failed.

Reply:
I don't see `onSuccess` callback here.

Reply:
it doesn`t work when i add onSuccess callback